### PR TITLE
[GH-692] Prepare release version

### LINKED ISSUE #692: Prepare production release

### PR DESCRIPTION
**Pull request by thepiwo**
resolves #692 

I will propose a follow up task to actually stage a release with some documentation

### CODE REVIEW

**Comment by thepiwo** [comment]
@cytadela8 @d-velev could you have a look at https://github.com/aeternity/elixir-node/pull/698/files#diff-6b3bd680cf7bd74a2ff8a2a4a449940eR24 and see if this can be solved in a better way?

**Comment by thepiwo** [comment]
@cytadela8 @d-velev I did resolve it, but you can still suggest a nicer way if you see one, eventually using the supervision trees

**Comment by thepiwo** [comment]
@cytadela8 @d-velev @DanielaIvanova can you review?

**Comment by thepiwo** [comment]
a binary release can be built with `VERSION=0.1.0 make prod-release` it will tell you in the output, how to execute.

File: rel/config.exs [hunk 1]
```diff
@@ -19,17 +19,13 @@ environment :prod do
 end
 
 release :elixir_node do
-  set version: "0.1.0"
+  set version: System.get_env("VERSION")
```
Comment (d-velev) [inline rel/config.exs]: what's the purpose of this?

Comment (thepiwo) [reply rel/config.exs]: to set version number as environment while releasing, to not have to change it manually in this file

Comment (d-velev) [reply rel/config.exs]: got it, thanks

**Comment by thepiwo** [comment]
@d-velev @DanielaIvanova can you re-approve?

**Pull request merged**